operations/restrictions.md

```diff
@@ -369,3 +375,113 @@ Removes restrictions from the service.
 ```javascript
 {}
 ```
+
+## Set restrictions
+
+Adds new restrictions with the specified conditions.
+
+If there already exists a restriction with the same conditions, following scenarios apply:
+
+- A. If the exceptions of the new restriction match the old restriction:
+   1) A If the new interval is longer than the old one, new restriction is created joining the two intervals.
+   2) If the new interval is shorter, no changes are made.
+- B. If the exceptions of the new restriction do not match the old restriction:
+   1) If the new interval overlaps the old interval, the old restriction will be spliced before and after the new interval. Restrictions matching the old restriction are then added at the appropriate interval along with the new restriction.
+   2) If the new interval do not overlap the old interval, the new restriction is added as usual.
+
+If the supplied restrictions match in all the properties but differ in interval and follow each other chronologically, the supplied restrictions will be joined into a single restriction.
+
+The usage of this endpoint must be enabled per enterprise by Connectivity team.
```

Comment:
I don't think this is information for integration partners :D They have just TPMS to communicate with.